CONTRIBUTING.md

```diff
@@ -76,12 +78,19 @@ Pull Requests will be automatically labled as "**Idle**" when:
 - No Engagement from the author of the PR to address Review Requests or input from the community for more than 15 calender days.
   - Personal holidays or public holidays will not be counted.
 - One or more identified Reviewers (including WSL) are not providing feedback/resolutions or approving the Pull Request for more than 15 calender days.
-- No consensus are reached during approval process.
 
 Any Pull Requests that are labeled as "Idle" will be discussed during Technical Steering Meeting to take a decision on, such as:
 - Close the PR and label it as "**Returned**".
 - "**Force**" Merging of the PR without having full reviewers approvals due to their inactivity.
-- Follow Governance Procedures to get consensus via agreement or voting mechanisms.
+
+<a name="unresolved"></a>
+### Unresolved Pull Requests Policy
+
+Pull Requests will be automatically labled as "**Unresolved**" when no consensus are reached during approval process.
```

Comment:
Can't be done automatically I suppose.

Reply:
When I say automatically, I mean either find a way to do it in GitHub Workflow or TSC will be manually applying the label (automatically from the sense that they are not applied but the author or WSL).